table_schemas/shiny90_survey/1_shiny90_survey_estimates.json

```diff
@@ -0,0 +1,111 @@
+{
+  "fields": [{
+      "name": "Country or region",
+      "title": "Country or region",
+      "description": "Country or region matching exactly the country or region in top left panel.",
+      "type": "string",
+      "constraints": {
+        "required": true
+      }
+    }, {
+        "name": "Survey Id",
+        "title": "Survey ID",
+        "description": "A unique identifier for the survey (each survey must have a unique name)",
+        "type": "string",
+        "constraints": {
+          "required": true
+        }
+    },
+    {
+      "name": "Year",
+      "title": "Year",
+      "description": "The year in which survey was conducted; year of survey fieldwork midpoint if survey spanned multiple years.",
+      "type": "integer",
+      "constraints": {
+          "required": true,
+          "minimum": 1970,
+          "maximum": 2022
+      }
+    },
+    {
+      "name": "Age Group",
+      "title": "Age Group",
+      "description": "The age group. Must be \"15-24\", \"25-34\", \"35-49\", \"50+\", \"15-49\" or \"15+\".",
+      "type": "string",
+      "constraints": {
+        "required": true,
+        "enum": ["15-24", "25-34", "35-49", "50+", "15-49", "15+"]
```

Comment:
Just to make sure, we're not using here our agreed age group format : `["Y000_014", "Y015_999", "Y000_999"]`

Reply:
This is something different to Naomi...  Shiny90 has it's own age group codes. Always good to check though.